README.md

```diff
@@ -321,6 +341,29 @@ Shorthand for `config.storeConfig.path`.
 
 Additional options to be passed to the `store`'s constructor.
 
+#### `middlewares`
+
+* Environment: `CONNECTOR_MIDDLEWARES`
+* Type: `array`
+* Default: `[]`
+
+| Name           | Type    | Description                                                              |
+| -------------- | ------- | ------------------------------------------------------------------------ |
+| `[]`           | object  | Object describing middleware instance.                                   |
+| `[].type`      | string  | NPM module that should be `require`d to load the middleware constructor. |
+| `[].priority`  | integer | Priority at which this middleware should be inserted. Must be unique.    |
```

Comment:
Is this field implemented? I don't see it used in the code.

Reply:
Nope, it was from a previous version, I'll take it out!

Reply:
Removed.